include/circt/Dialect/RTL/Structure.td

```diff
@@ -32,6 +32,12 @@ def RTLModuleOp : RTLOp<"module",
     // squatting on the name.
     Block *getBodyBlock() { return &body().front(); }
 
+    // Get the module's symbolic name.
+    StringRef getName() {
```

Comment:
Feel free to land things like this as an independent patch if it is helpful.

Reply:
:+1: will do. In this case it all kind of flowed together, but I will try to make some small pre-emptive additions like this to cut down on review complexity.